src/gui/mzroll/mainwindow.cpp

```diff
@@ -3892,27 +3892,9 @@ void MainWindow::markGroup(PeakGroup* group, char label) {
 	//getPlotWidget()->scene()->update();
 }
 
-int MainWindow::versionCheck() {
-
-//define a MACRO for converting DEFINES to strings.. jfc!
-#define xstr(s) str(s)
-#define str(s) #s
-
-	QString hostname = "http://genomics-pubs.princeton.edu";
-	QString path = "/mzroll/vercheck.php?";
-	QString os = "os=" + QString(xstr(PLATFORM));
-	//QString ver = "ver=" + QString::number(EL_MAVEN_VERSION);
-	QString ver = "ver=620";
-	QString query = os + "&" + ver;
-
-	logWidget->append("Latest version check: " + hostname + path + query);
-
-	QDownloader* downloader = new QDownloader(this);
-	downloader->getPage(hostname + path + query);
-	connect(downloader, SIGNAL(downloadResult(QString)), logWidget,
-			SLOT(append(QString)));
-
-	return 0;
+QString MainWindow::appVersion() {
```

Comment:
Can we please make this `static` ? There are  places where we need to get the app version without actually requiring the entire mainwindow instance

Reply:
Yes I can do that.